src/modm/board/nucleo_l432kc/board.hpp

```diff
@@ -67,7 +67,7 @@ struct SystemClock {
 	}
 };
 
-// Arduino Nano Footprint
+// Arduino Uno Footprint
```

Comment:
all the *k identifiers point to Nucleo32

Reply:
I've just copied this... so already ctrl+h't all of these ;)

Reply:
I… don't understand your response?

- [Nucleo32 has Arduino™ Nano V3 connectivity](https://www.st.com/content/st_com/en/products/evaluation-tools/product-evaluation-tools/mcu-mpu-eval-tools/stm32-mcu-mpu-eval-tools/stm32-nucleo-boards/nucleo-g031k8.html).
- [Nucleo64 has ARDUINO® Uno V3 connectivity](https://www.st.com/content/st_com/en/products/evaluation-tools/product-evaluation-tools/mcu-mpu-eval-tools/stm32-mcu-mpu-eval-tools/stm32-nucleo-boards/nucleo-f072rb.html).